Microsoft.Toolkit.Win32.UI.Controls/WPF/WebView/WebViewControlProcessExtensions.cs

```diff
@@ -31,7 +34,7 @@ internal static class WebViewControlProcessExtensions
         /// <paramref name="hostWindowHandle"/> is equal to <see cref="IntPtr.Zero"/>, or
         /// <paramref name="process"/> is <see langword="null" />.
         /// </exception>
-        internal static IWebView CreateWebView(
+        public static IWebView CreateWebView(
```

Comment:
This looks like an entire helper-class for a single method call: `return new WebView(process.CreateWebViewControlHost(hostWindowHandle, bounds));`
Why?
If it should be a helper, why not help with the handle, bounds etc?

Reply:
Aaaah I missed that. Makes sense.

> I'm not sure if they want to use the HWND of the Window, or a control deriving from HWNDHost

You'd be sure if there were different overloads that each took the Window, HwndHost or WinForms control

Reply:
I'm looking into adjusting the API now (adding more overloads) so that it's much easier for the WPF developer. Would we want to pass in Window, VisualElement, and HWndHost and try to figure it out from there?

Reply:
Probably. But you likely want to keep the handle overload too?

Reply:
Yes, I'm working on a prototype now that takes a Visual and Rect, Window and Rect, (and overloads without the Rect to fill all available space determined by ActualHeight/ActualWidth). Makes most of this pretty easy.  Any sane way to ensure STA while creating the control as in the example above? Right now I'm having to put the code into a thread and manually set the apartment state.

Reply:
I pushed an update that has some helpers that add value to make WebView fairly simple to use in or out of a designer.